packages/core/src/PartialExecutionUtils/__tests__/filterDisabledNodes.test.ts

```diff
@@ -0,0 +1,124 @@
+// NOTE: Diagrams in this file have been created with https://asciiflow.com/#/
```

Comment:
I just moved the tests from the other file here. It's the same setup, just a different function call.